plugins/spz/io/SpzWriter.cpp

```diff
@@ -41,24 +42,25 @@
 namespace pdal
 {
 
-static StaticPluginInfo const s_info
+static PluginInfo const s_info
 {
         "writers.spz",
         "SPZ writer",
-        "http://pdal.io/stages/writers.spz.html",
-        { "spz" }
+        "http://pdal.io/stages/writers.spz.html"
 };
 
-CREATE_STATIC_STAGE(SpzWriter, s_info)
+CREATE_SHARED_STAGE(SpzWriter, s_info)
 
-SpzWriter::SpzWriter() : m_cloud(new spz::PackedGaussians)
+SpzWriter::SpzWriter() : m_cloud(new spz::GaussianCloud)
 {}
 
 std::string SpzWriter::getName() const { return s_info.name; }
 
 void SpzWriter::addArgs(ProgramArgs& args)
 {
     args.add("antialiased", "Mark the data as antialiased", m_antialiased);
+    args.add("input_orientation", "Coordinate system of points being written; default is 'RUB'. "
```

Comment:
We have `input_orientation` on `writers.spz` and `out_orientation` on `readers.spz`. Besides the inconsistent naming, why does one say _input_ on something that writes data and the other says _out_ on something that reads it? 

I think should just always pick an assumed orientation when we read data, always use that internally, and then provide a single option on the writer if someone wants to flip it. The dual-mode thing is going to be super confusing for the user because they have no way to know the orientation of the points in the PointView.